app/__tests__/PackageDetailPage.test.js

```diff
@@ -10,7 +10,7 @@ const mockPackageData = {
   identifier: "f78742e5-6af9-4756-a94a-6cd297406d51",
   title: "Test Package",
   origin: "aurora",
-  last_outcome: "SUCCESS",
+  status: "SUCCESS",
```

Comment:
This should be COMPLETE or IN PROGRESS now, right?

Reply:
Updated!